context/app/static/js/components/Search/AncestorNote.jsx

```diff
@@ -4,17 +4,17 @@ import Typography from '@material-ui/core/Typography';
 
 import { LightBlueLink } from 'js/shared-styles/Links';
 
-function AncestorNote(props) {
-  const { entity } = props;
-
+function AncestorNote({ entity, ancestorOrDescendantNote }) {
   let message = '...';
+
+  const text = ancestorOrDescendantNote === 'ancestor' ? 'Derived from' : 'Ancestor of';
```

Comment:
```suggestion
function AncestorNote({ entity, isDerivedFrom }) {
  let message = '...';

  const text = isDerivedFrom ? 'Derived from' : 'Ancestor of';
```
Or make the default the other way? Not sure which is best, but the string equality check makes me nervous.

Reply:
As I mentioned above, this isn't finished and will be reworked.